src/crypto/key.ml

```diff
@@ -57,3 +57,13 @@ let encoding =
 
 let to_yojson, of_yojson =
   Yojson_ext.with_yojson_string "key" to_string of_string
+
+include Helpers.Bin_prot_ext.String_like (struct
+  type nonrec t = t
+
+  let name = "Key"
```

Comment:
Does `Key` with capital matter? because the `key_hash` in `key_hash.ml` where name is not capitalize?